cluster-sync/imageio/imageio.yaml

```diff
@@ -22,7 +22,8 @@ spec:
         ports:
         - containerPort: 12345
       - name: fakeovirt
-        image: machacekondra/fakeovirt:test
+        # update back to original image when changes are merged
+        image: quay.io/jdzon/fakeovirt:latest
```

Comment:
I've updated the repo with your changes, thanks.

Reply:
Thanks!